### DISCUSSION TITLE
Can you update dependencies of @wagmi/core "zustand"

### DISCUSSION THREAD

**Discussion by ibweb3dev** [wevm/wagmi#1748] [Ideas]
"zustand": "^4.1.4" is outdated and the newer version fixes various deprecated 

[DEPRECATED] default export is deprecated, instead import { shallow } from'zustand/shallow'

**Comment by tmm** [comment]
The most recent version of wagmi uses [`zustand@4.3.1`](https://github.com/wagmi-dev/wagmi/blob/main/packages/core/package.json#L114).
